backend/src/routes/__test__/encounter.route.test.ts

```diff
@@ -366,6 +376,54 @@ describe('POST /encounter', () => {
     })
 })
 
+describe('GET /encounters/:id', () => {
+    it ('Successfully retrieves an encounter with a given json', async () => {
+        const newEncounterId = await createUserPersonEncounter(token);
```

Comment:
Good that you figured out that you needed createUserPersonEncounter - I was gonna tell you but needed to head off haha